GRASS community map (#169)
OSGeo/grass-website: static/geojson/README.md

> # GRASS GIS map

Reviewer: ```suggestion
# GRASS community map
```

> In this folder you can find geographical data related to GRASS GIS community. The data are provided in GeoJSON format.

Reviewer: ```suggestion
In this folder you can find geographical data related to the GRASS community. The data are provided in GeoJSON format.
```

## User map

> The [user.geojson file](https://github.com/OSGeo/grass-website/static/geojson/user.geojson) contain the position of GRASS GIS users.

Reviewer: ```suggestion
The [user.geojson file](https://github.com/OSGeo/grass-website/static/geojson/user.geojson) contains the positions of GRASS users and community members who willingly added themselves.
```

The [provider.geojson file](https://github.com/OSGeo/grass-website/static/geojson/provider.geojson) contain the position of any companies,

> research center, university providing activities (specially development and courses) related to GRASS GIS.

Reviewer: ```suggestion
The [provider.geojson file](https://github.com/OSGeo/grass-website/static/geojson/provider.geojson) contains the position of companies,
research centers and universities offering support related to GRASS, i.e., commercial support, new features development and implementations, courses, etc.
```

> ## Add new record

Reviewer: ```suggestion
## Add new records to the map
```

> If you want to add a new record to one of the GeoJSON files you have to ways

Reviewer: ```suggestion
```

### User without Git/Github experience

> * You need to create a [Github account](https://github.com/join)

Reviewer: ```suggestion
* Create a [Github account](https://github.com/join)
```

* Sign in and move to https://github.com/OSGeo/grass-website/ repository

> * Choose the file to modify into [GeoJSON files](https://github.com/OSGeo/grass-website/static/geojson/)

Reviewer: ```suggestion
* Choose which file to modify from the [GeoJSON files](https://github.com/OSGeo/grass-website/static/geojson/)
```

* Click the pencil button to modify the file
* Add a new feature copying the feature template in the bottom of the page 

> * When you save the file, select `Create a new branch for this commit and start a pull request.`

Reviewer: ```suggestion
* When saving the file, select `Create a new branch for this commit and start a pull request.`
```

# GRASS community map

In this folder you can find geographical data related to the GRASS community.
The data are provided in GeoJSON format.

## User map

> The [user.geojson file](https://github.com/OSGeo/grass-website/static/geojson/user.geojson)

Reviewer 2: ```suggestion
The [user.geojson file]((https://github.com/OSGeo/grass-website/blob/master/static/geojson/user.geojson)
```

contains the positions of GRASS users and community members who willingly
added themselves.

> The [provider.geojson file](https://github.com/OSGeo/grass-website/static/geojson/provider.geojson)

Reviewer 2: ```suggestion
The [provider.geojson file](https://github.com/OSGeo/grass-website/blob/master/static/geojson/provider.geojson)
```

contains the position of companies, research centers and universities
offering support related to GRASS, i.e., commercial support, new features
development and implementations, courses, etc.

## Add new records to the map

### User without Git/Github experience

* Create a [Github account](https://github.com/join)
* Sign in and move to <https://github.com/OSGeo/grass-website/> repository

> * Choose which file to modify from the [GeoJSON files](https://github.com/OSGeo/grass-website/static/geojson/)

Reviewer 2: ```suggestion
* Choose which file to modify from the [GeoJSON files](https://github.com/OSGeo/grass-website/tree/master/static/geojson/)
```
